### ISSUE TITLE
[FEATURE] Custom Time/Date formats for Lawnchair's At a Glance

### ISSUE THREAD

**Issue by Username1-a** [Deletescape-Media/Lawnchair#4246]
### Describe the feature

Ability to choose custom time and date from the predefined list or specify custom format string

Also, ability to change format to Titlecase/Uppercase/Lowercase

### Why would it be useful to add?

More customization and choice of displaying information 

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

Examples:
![Screenshot_20240408-125529_Smartspacer](https://github.com/LawnchairLauncher/lawnchair/assets/47947180/ed621d50-65ef-4b95-8a8e-271f0a46003a)

![Screenshot_20240408-125517_Smartspacer](https://github.com/LawnchairLauncher/lawnchair/assets/47947180/dcceb619-8d46-4f4e-a563-9edefe843ad7)

**Comment by meichthys** [comment]
I would agree with this. The current format of "November 18 at 9:15am" doesn't really make sense in english. 
It should also be possible to disable both the date and time. Currently only one can be disabled at a time.
